spec/services/events/validate_creation_service_spec.rb

```diff
@@ -227,6 +227,35 @@
           end
         end
       end
+
+      context 'when event belongs to a recurring persisted event and subscription is terminated' do
+        let(:billable_metric) do
+          create(
+            :billable_metric,
+            organization:,
+            aggregation_type: 'recurring_count_agg',
+            field_name: 'item_id',
+          )
+        end
+
+        let(:subscription) { create(:subscription, customer:, organization:, status: :terminated) }
```

Comment:
can we put the one line `let` before the blocks?

Reply:
why this convention? 😅 - I personally prefer to put them to be coherent when it's used (here for instance, subscription is called when we're using the initializing `params`)

Reply:
it's only for readability 😂